solutions/14.js

```diff
@@ -0,0 +1,17 @@
+// Rahul Kalra
+// Return an array of the diagnal values starting from 0,0 from a 2D array input(gauranteed to be square).
```

Comment:
@rkalra247, please make sure there is no typo in comment, and add comments like this: https://github.com/llipio/algorithms/pull/55/files